.github/workflows/run_tests_matlab.yaml

```diff
@@ -0,0 +1,68 @@
+name: MATLAB tests
+
+# Installs
+# - MATLAB github action
+# - MOXunit
+# - MOcov
+# Get test data
+# cd into .github/workflows
+# run .github/workflows/tests_matlab.m
+# If tests pass, uploads coverage to codecov
+
+on:
+  push:
+    # TODO only run on master branch on push 
+    branches: ["*"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  tests:
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest] # "macos-latest" or "windows-latest" don't work (yet?)
+        matlab-version: ["R2020a"] # add more versions if needed: "R2021a"
+      fail-fast: false  # Don't cancel all jobs if one fails
```

Comment:
If we ever wanted to test on different OS or matlab version this could simply be changed in there.

Currently Matlab github actions only support linux.